Directory.Build.props

```diff
@@ -0,0 +1,66 @@
+﻿<Project>
+    <PropertyGroup>
+        <Copyright>Copyright (c) Seth Reno. All rights reserved</Copyright>
+        <PackageLicenseExpression>MIT</PackageLicenseExpression>
+        <PackageProjectUrl>https://github.com/sethreno/ReGitLint</PackageProjectUrl>
+        <Authors>Seth Reno</Authors>
+        <Owners>Seth Reno</Owners>
+        <PackageTags>.net;c#;git;lint;ci</PackageTags>
+        <PackageReleaseNotes>https://github.com/sethreno/ReGitLint/releases</PackageReleaseNotes>
+        <PackageReadmeFile>readme.md</PackageReadmeFile>
+        <PackageRequireLicenseAcceptance>false</PackageRequireLicenseAcceptance>
+        <NeutralLanguage>en-US</NeutralLanguage>
+        <MinClientVersion>2.14</MinClientVersion>
+
+        <PublishRepositoryUrl>true</PublishRepositoryUrl>
+        <RepositoryUrl>https://github.com/sethreno/ReGitLint</RepositoryUrl>
+        <RepositoryType>git</RepositoryType>
+        <Product>$(AssemblyName) ($(TargetFramework))</Product>
+        <Version>6.1.0-alpha</Version>
+
+        <!--<Nullable>enable</Nullable>-->
+        <ImplicitUsings>enable</ImplicitUsings>
+        <LangVersion>latest</LangVersion>
+        <NoWarn>$(NoWarn);CS1591</NoWarn>
+        <Platform>AnyCPU</Platform>
+
+        <EnableNETAnalyzers>true</EnableNETAnalyzers>
+        <AnalysisLevel>latest</AnalysisLevel>
+        <AnalysisMode>Default</AnalysisMode>
+
+        <IsTestProject>$(MSBuildProjectName.Contains('Test'))</IsTestProject>
+    </PropertyGroup>
+
+    <PropertyGroup Condition=" '$(Configuration)' == 'Release' ">
+        <!--<SymbolPackageFormat>snupkg</SymbolPackageFormat>-->
+        <IncludeSymbols>true</IncludeSymbols>
+        <AllowedOutputExtensionsInPackageBuildOutputFolder>$(AllowedOutputExtensionsInPackageBuildOutputFolder);.pdb</AllowedOutputExtensionsInPackageBuildOutputFolder>
+        <EmbedUntrackedSources>true</EmbedUntrackedSources>
+
+        <DesignTimeBuild>false</DesignTimeBuild>
+        <DebugType>portable</DebugType>
+        <DebugSymbols>true</DebugSymbols>
+
+        <GenerateDocumentationFile>true</GenerateDocumentationFile>
+        <GeneratePackageOnBuild Condition=" '$(IsTestProject)' != 'true'">true</GeneratePackageOnBuild>
+        <PackageIcon>icon.png</PackageIcon>
+        <ContinuousIntegrationBuild>true</ContinuousIntegrationBuild>
+        <Deterministic>true</Deterministic>
+    </PropertyGroup>
+
+    <PropertyGroup Condition=" '$(Configuration)' == 'Debug' ">
+        <DebugType>full</DebugType>
+        <DebugSymbols>true</DebugSymbols>
+    </PropertyGroup>
```

Comment:
Isn't this the default behavior?

Reply:
DebugType: see https://stackoverflow.com/questions/46902377/what-is-the-difference-between-using-debugtypefull-debugtype-and-debugtype:
> The difference is that the "full" type emits a classic windows PDB symbol file which is complex and poorly documented. The "portable" PDB format is a new open-source format that can be created and used on all platforms.

By not explicitly specifying these, the proper defaults are used, depending on the project type, configuration, target runtime etc. The only case I'm aware of to force "full", is to make some code coverage tools work, which does not apply here.

DebugSymbols: I'm pretty sure these are on for debug builds. Otherwise you wouldn't get .pdb files.